### DISCUSSION TITLE
Q: Where is data stored on Yeti?

### DISCUSSION THREAD

**Discussion by deepti4666** [YetiForceCompany/YetiForceCRM#16386] [How to]
Where is data stored on Yeti, and in which file & folder on the server?

1. I created a new installation
2. With a new database
3. Used the "Replace Into" command to copy data from one database to the new database
4. Recovered the files from the backup

**Problem:**
No data showing

How do I find the data that ware there?

**Comment by rskrzypczak** [comment]
Check _config/_ folder 
In particular: config/Db.php, config/Main.php

**Reply by deepti4666** [reply]
Do I change the database settings there to the old database?
